### DISCUSSION TITLE
Toggle Embedding Search

### DISCUSSION THREAD

**Discussion by srijrao** [brianpetro/obsidian-smart-connections#272] [Ideas]
This app is great, but I need the option to turn off the searching of my notes. Sometimes I just want to speak to plain ChatGPT, not my notes.

**Comment by brianpetro** [comment]
Hi @srijrao 

Currently, there isn't an option to turn off the searching of your notes. 

However, you can still use ChatGPT without referencing your notes by simply not including any prompts related to your notes. The words to avoid are "self-referential" pronouns. In english, these are: `my, I, me, mine, our, ours, us, we`

When you avoid these words the model will focus solely on generating responses based on the provided input. 

Thanks for using Smart Connections
Brian  🌴

**Reply by wafe** [reply]
Sometimes I can't avoid self-referential keywords when I am trying to clean up the nuances of an email. How about ignoring parts wrapped in code block ``` from self-referential keyword searches?

**Reply by wafe** [reply]
I also tried to circumvent it by setting the default language to 'it', but that didn't work because all the responses were in Italian.

**Comment by orand** [comment]
So if I ask "when is Brian's birthday", it won't use my notes, but if I ask "when is my buddy Brian's birthday", it will?

**Reply by brianpetro** [reply]
It has to do with the note retrieval being triggered by self-referential pronouns

🌴

**Reply by orand** [reply]
Thanks, so the answer is yes. Have you found a consistent short "cheater" way to always trigger this in situations where a self-referential pronoun wouldn't otherwise naturally occur?

**Reply by brianpetro** [reply]
@orand I have some ideas for ways to improve that triggering, but at the moment the self-referential keyword trigger is the only way for a broad search.

However, if you're target is a specific folder or specific note, you can trigger the lookup by including a folder input (type `/` to select a folder) or link to note (type `[[` to select a note, uses standard Obsidian internal link format). When either is included, a lookup/search will be performed similar to as if your included a self-referential pronoun.
